gui/default/syncthing/core/syncthingController.js

```diff
@@ -1661,6 +1661,26 @@ angular.module('syncthing.core')
             $http.post(urlbase + "/db/scan?folder=" + encodeURIComponent(folder));
         };
 
+        $scope.pauseAllFolders = function () {
+            // loop through all folders and pause them
+            var folderListCache = $scope.folderList();
+            for (var i = 0; i < folderListCache.length; i++) {
+                if (!folderListCache[i].paused) {
+                    $scope.setFolderPause(folderListCache[i].id, true);
```

Comment:
If you have 100s of folders, this will be very inefficient.
If you have a 100 folders, it will post the config, emit events, reload half of the services for each folder.
If you have 100s of folders, this will be very inefficient.
If you have a 100 folders, it will post the config, emit events, reload half of the services for each folder.